### ISSUE TITLE
PHP cURL is not installed

### ISSUE THREAD

**Issue by ghost** [YunoHost-Apps/rainloop_ynh#36]
Hi.

I got a server on Debian 8 "Jessie" through Gandi IaaS. I installed YunoHost and then tried to install Rainloop (no other app was installed), but it failed.
I used `yunohost app install rainloop --verbose`. I found that:

```
<p>[302] The following PHP extensions are not available in your PHP configuration!</p><ul><li>cURL</li></ul>+ EXIT_PROPERLY
+ exit_code=46
+ '[' 46 -eq 0 ']'
+ trap '' EXIT
+ set +eu
+ echo -e '\e[91m \e[1m'
 
+ echo -e '!!\n  rainloop install'\''s script has encountered an error. Installation was cancelled.\n!!'
+ type -t CLEAN_SETUP
```

I installed the PHP extension for cURL with package [php5-curl](https://packages.debian.org/jessie/php5-curl). For Debian Stretch and more, [php-curl](https://packages.debian.org/stable/php-curl) would be the good one.
Then I tried again to install Rainloop and it works. Where should be done the installation (`scripts/install`, `manifest.json`, somewhere else) ?

Regards.

It seems to be also the problem of: https://github.com/YunoHost-Apps/rainloop_ynh/issues/32

**Comment by buzuck** [comment]
I confirm this issue, and the need to install php-curl for Stretch.

RainLoop needs the `dom` extension as well, provided by [`php-xml`](https://packages.debian.org/en/stretch/php/php-xml).

----

There is a list of the needed modules on the [requirement's page](http://www.rainloop.net/docs/system-requirements/) :  

> PHP extensions: cURL, iconv, json, libxml, dom, openssl, DateTime, PCRE, SPL

**Comment by pidgy** [comment]
I fixed this problem by restarting my apache service,

`   service apache2 restart`

**Comment by SohKa** [comment]
FYI, when trying to install Rainloop, it first failed because of this : 

`2019-03-01 13:25:01,220: DEBUG - <p>[302] The following PHP extensions are not available in your PHP configuration!</p><ul><li>cURL</li></ul>+ EXIT_PROPERLY`

Like described in this issue, I just had to install `php-curl` (`sudo apt install php-curl`). Then the installation failed again because of this : 

`2019-03-01 13:26:40,044: DEBUG - <p>[302] The following PHP extensions are not available in your PHP configuration!</p><ul><li>dom</li></ul>+ EXIT_PROPERLY`

So I installed `php-xml` (`sudo apt install php-xml`). Finally, the installation went through.

**Comment by ericgaspar** [comment]
Should be fixed with #68
